Orlando Walking Tours/Services/InMemoryModelService.swift

```diff
@@ -22,10 +22,10 @@ class InMemoryModelService : ModelService {
         let cityTour = self.tours[1]
         // add some locations to tour
         // when printed to verify why were first two in reverse order?
-        self.addLocation(location: locations[0], toTour: cityTour, completion: nil)
-        self.addLocation(location: locations[2], toTour: cityTour, completion: nil)
-        self.addLocation(location: locations[3], toTour: cityTour, completion: nil)
-        self.addLocation(location: locations[5], toTour: cityTour, completion: nil)
+        self.add(locations[0], to: cityTour, completion: nil)
```

Comment:
probably should use a for loop for each location that way no accidentally out of bound crashes.